src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final double base;
+    private final double height;
```

Comment:
Follow one code style. Params in all inherited classes should be final or not, choose one of these options and set it for all classes